pkg/operator/helpers.go

```diff
@@ -253,6 +254,19 @@ func GetCSVBase(name, namespace, displayName, description, image, replaces strin
 				"storage_class": "host-nfs-storageclass",
 			},
 		},
+		map[string]interface{}{
+			"apiVersion": "openstackclients.controlplane.openstack.org",
```

Comment:
Change it to this perhaps? controlplane.openstack.org/v1beta1

Reply:
Done.